lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -391,6 +392,39 @@ def allocate_load_balancer(action_handler, lb_spec, lb_options, machine_specs)
         end
       end
 
+      # Update the load balancer sticky sessions
+      if sticky_sessions
+        policy_name = "#{actual_elb.name}-sticky-session-policy"
+        policies = elb.client.describe_load_balancer_policies(load_balancer_name: actual_elb.name)
+        current_cookie_hash = cookie_from_policies(policies)
+
+        if current_cookie_hash[:attribute_value] != sticky_sessions[:cookie_name]
+          perform_action.call("  updating sticky sessions to #{sticky_sessions[:cookie_name].inspect}") do
+            count = 0
+
+            begin
+              elb.client.create_app_cookie_stickiness_policy(
+                load_balancer_name: actual_elb.name,
+                policy_name: policy_name,
+                cookie_name: sticky_sessions[:cookie_name]
+              )
+            rescue AWS::ELB::Errors::DuplicatePolicyName
```

Comment:
Funny thing about this error.  When I was testing this stuff in `irb`, I was able to call `elb.client.create_app_cookie_stickiness_policy` repeatedly, and it appeared to be idempotent, in that no errors were raised and the stickiness policy remained unchanged.  However, when I ran this in the context of the specs, and it was testing whether it was idempotent or not (via the `be_idempotent` matcher), this error was thrown.  So as best I can tell, it received it when trying to create a stickiness policy with the same parameters as one that already existed.